go.mod

```diff
@@ -4,7 +4,7 @@ go 1.16
 
 require (
 	github.com/fatih/color v1.9.0 // indirect
-	github.com/go-gandi/go-gandi v0.0.0-20211019080925-298766c2d6b7
+	github.com/go-gandi/go-gandi v0.0.0-20211203125005-d06517bcd2ac
```

Comment:
Ref: https://github.com/go-gandi/go-gandi/commit/d06517bcd2ac06a0f6459a57f6ac51b78075c6fe

Reply:
Updated to library integrating Get request.